src/hooks/useList.js

```diff
@@ -0,0 +1,30 @@
+const { MIN_PREV_TRACKS_TO_SHOW, MAX_NEXT_TRACKS_TO_SHOW } = require('@consts');
+let listState = {
+    MIN_PREV_TRACKS: MIN_PREV_TRACKS_TO_SHOW,
+    MAX_NEXT_TRACKS: MAX_NEXT_TRACKS_TO_SHOW,
+}
+
```

Comment:
it's must be a class 'cause we will add more functionality to list like pagiantion and so